README.md

```diff
@@ -7,22 +7,24 @@
 
 # Nancy
 
-`nancy` is a tool to check for vulnerabilities in your Golang dependencies, powered by [Sonatype OSS Index](https://ossindex.sonatype.org/).
+`nancy` is a tool to check for vulnerabilities in your Golang dependencies, powered by [Sonatype OSS Index](https://ossindex.sonatype.org/), and as well, works with Nexus IQ Server, allowing you a smooth experience as a Golang developer, using the best tools in the market!
 
 ### Usage
 
 ```
  ~ > nancy
 Usage:
-go list -m all | nancy [options]
-nancy [options] </path/to/Gopkg.lock>
-nancy [options] </path/to/go.sum>
+        nancy [options] </path/to/Gopkg.lock>
```

Comment:
why did this option get deleted?? `go list -m all | nancy [options]`

Reply:
GOOD QUESTION.